server/tests/misc.spec.js

```diff
@@ -17,13 +18,13 @@ describe('## Misc', () => {
     });
 
     describe('# GET /api/404', () => {
-        xit('should return 404 status', () => request(app)
+        it('should return 404 status', () => request(app)
             .get('/api/404')
+            .set('Authorization', `Bearer ${auth}`)
             .expect(httpStatus.NOT_FOUND)
             .then((res) => {
-                expect(res.body.code).to.equal('UNKNOWN_API');
-                expect(res.body.status).to.equal('ERROR');
-                expect(res.body.message).to.equal('API not found');
+                expect(res.status).to.equal(404);
+                expect(res.body.message).to.equal('Not Found');
```

Comment:
Should we check `res.body.message` here or should we check the `res.body.code` ? I think the `res.body.code` is much less likely to change.